docs/porting/connectivity/ble.md

```diff
@@ -0,0 +1,37 @@
+# Bluetooth Low Energy (BLE)
+
+Mbed OS uses the Cordio stack for its BLE host. We recommend porting the Cordio platform<!--is that what they call it?--> to your board, or porting at the HCI level<!--what's the distinction between porting the platform and porting the HCI level? If one is ported to the board, where is the other ported to? The chip? What's the difference between porting to a whole board v porting to a chip?--> as described in the HCI documentation.
+<!--we need a decision from Andy about the Cordio docs before we start linking. Ours are 18 months old!-->
+
+## Replacing the Cordio stack
+
+You can replace the Cordio stack by porting a complete alternative stack. You must do this at the API level<!--as opposed to which level?--> by:
```

Comment:
```suggestion
You can replace the Cordio stack by porting a complete alternative stack. You must port the full API, not just the HCI, by:
```

Reply:
that is somewhat misleading, if you're porting the API then HCI is completely irrelevant and down to your implementation.

Reply:
Is this one okay now?